superset/tags/commands/create.py

```diff
@@ -96,13 +94,13 @@ def run(self) -> None:
     def validate(self) -> None:
         exceptions = []
         # Validate object_id
-        if self._objects_to_tag:
-            if any(obj_id == 0 for obj_type, obj_id in self._objects_to_tag):
+        if objects_to_tag := self._properties.get("objects_to_tag", []):
+            if any(obj_id == 0 for obj_type, obj_id in objects_to_tag):
```

Comment:
Why are we checking for id 0 here? Is this something that could happen realistically? Why not check for negative numbers, or integers — can we move this validation to the Marshmallow schema?